graylog2-server/src/main/java/org/graylog2/alerts/types/FieldValueAlertCondition.java

```diff
@@ -175,4 +178,9 @@ protected CheckResult runCheck() {
             return new NegativeCheckResult(this);
         }
     }
+
+    @Override
+    public List<Message> getSearchHits() {
+        return searchHits;
```

Comment:
I'm not sure if this class is ever used concurrently from multiple threads, but if it was, this should either return a copy of the list (`ImmutableList#copyOf()`) or change the `searchHits` field to a `CopyOnWriteArrayList`.